test/util.cpp

```diff
@@ -712,3 +712,56 @@ TEST(util, topological_sort_replace)
 
     EXPECT_TRUE(custom_sorter_used);
 }
```

Comment:
Need some bf16/fp16 cases, including with range out of bf16/fp16 non-infinity range.

Reply:
Done.